executor/recover_test.go

```diff
@@ -346,15 +345,20 @@ func TestRecoverClusterMeetError(t *testing.T) {
 	errorMsg = fmt.Sprintf("[ddl:-1]Detected TiDB upgrade during [%s, now), can't do flashback", oracle.GetTimeFromTS(nowTS).String())
 	tk.MustGetErrMsg(fmt.Sprintf("flashback cluster to timestamp '%s'", oracle.GetTimeFromTS(nowTS)), errorMsg)
 
-	require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/expression/injectSafeTS"))
-	require.NoError(t, failpoint.Disable("tikvclient/injectSafeTS"))
+	require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/ddl/injectSafeTS"))
 	require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/ddl/mockFlashbackTest"))
 }
 
 func TestFlashbackWithSafeTs(t *testing.T) {
 	store := testkit.CreateMockStore(t)
 	tk := testkit.NewTestKit(t, store)
 
+	originTimeout := ddl.FlashbackGetMinSafeTimeTimeout
+	ddl.FlashbackGetMinSafeTimeTimeout = 0
+	defer func() {
+		ddl.FlashbackGetMinSafeTimeTimeout = originTimeout
+	}()
```

Comment:
It is possible to use failpoint to pass value:

```
failpoint.Inject("failpoint-name", func(val failpoint.Value) {
                   t := time.Parse(val.(string))
                })
```

Just a suggestion, I don't like this global var much.

Reply:
Done, PTAL